vivisect/impemu/emulator.py

```diff
@@ -78,16 +79,24 @@ def __init__(self, vw, **kwargs):
                     Please note that since any possible effects the unsupported instruction are not
                     propagated, any possible flags or execution state are most likely misaligned from what
                     a real CPU might experience.
+        * loglevel
+            - Type: Integer
+            - Default: 30 (Corresponds to logging.WARNING)
+            - Desc: Because we do partial emulation for several of our heuristic passes like isProbablyCode,
+                    but also full emulation for several other analysis passes, there are several logging
+                    calls that are erroneous under the heuristic passes and typically we want those silenced
+                    under the heuristic passes only.
         '''
         self.vw = vw
-        self.funcva = None # Set if using runFunction
+        # Set down below in runFunction
+        self.funcva = None
         self.emustop = False
 
         self.hooks = {}
         self.taints = {}
         self.taintva = itertools.count(0x4156000F, 0x2000)
-        self.taintoffset = 0x1000
-        self.taintmask = 0xffffe000
+        self.taintoffset = 0x10
```

Comment:
i can't say i like this at all....  this large separation allows for tracking taints as they're modified, so we end up with <foo+352>.
what's the reason for this change?